bigtable/google/cloud/bigtable/row.py

```diff
@@ -245,7 +244,7 @@ def _retry_commit_exception(exc):
                             exceptions.DeadlineExceeded))
 
 
-class DirectRow(_SetDeleteRow):
+class DirectRow(mutation.RowMutations):
```

Comment:
What do you think about extending `_SetDeleteRow` from `mutation.RowMutations` instead of just doing it in `DirectRow`?

Reply:
@sduskis _SetDeleteRow is inherited by other classes in row.py:  ConditionalRow and AppendRow and RowMutations isn't integrated with these classes. Note that these classes use other GAPIC api's, not the GAPIC api used by RowMutations.